lib/util/streams.ts

```diff
@@ -0,0 +1,19 @@
+import { Blob } from 'buffer';
+import { Readable } from 'stream';
+import ReadableStream = NodeJS.ReadableStream;
+
+export async function streamToString(
+  stream: Readable | ReadableStream<any> | Blob
```

Comment:
Null-check provides the following error:
```
lib/util/streams.ts:6:22 - error TS2315: Type 'ReadableStream' is not generic.
```
This is how it's defined in the S3 library and it seems to work just fine. I don't know what to do to fix this. Please advise.

Reply:
You probably need to use the s3 variant of the stream

Reply:
Remove the generic type annotation, this also should be unified with this:

https://github.com/renovatebot/renovate/blob/982cefff2b8a3c697fe52fd5e023580a0678322e/lib/modules/platform/azure/util.ts#L121-L132

Reply:
> You probably need to use the s3 variant of the stream

I don't know how to fix this, sorry. Can you help me out? I don't see a specific S3 variant of that.